README.md

```diff
@@ -1,24 +1,163 @@
-# Sensu Go Remediation Handler
+# Sensu Remediation Handler
 
-The Sensu Go Remediation Handler is a simple [Sensu Go event handler][handler]
-for configuring self-healing workflows using Sensu. This handler was heavily
-inspired by [Nick Stielau's "Sensu Remediator"][remediator] (circa 2012).
+[![Bonsai Asset Badge](https://img.shields.io/badge/Bonsai-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-remediation-handler) [![Build Status](https://travis-ci.com/sensu/sensu-remediation-handler.svg?branch=master)](https://travis-ci.com/sensu/sensu-remediation-handler)
 
-The Sensu Go Remediation Handler &ndash; and other similar "self healing"
-workflows in Sensu &ndash; use a few simple Sensu features:
+- [Overview](#overview)
+- [Configuration](#configuration)
+  - [Environment Variables](#environment-variables)
+  - [Annotations](#annotations)
+    - [Annotation specification](#annotation-specification)
+    - [Remediation action specification](#remediation-action-specification)
+- [Setup](#setup)
+- [Usage Example](#usage-example)
+  - [Example "Unscheduled" Check (Remediation Action)](#example--unscheduled--check--remediation-action-)
+  - [Example Check Definition and Remediation Request Configuration](#example-check-definition-and-remediation-request-configuration)
+- [Acknowledgements](#acknowledgements)
+
+## Overview
+
+The Sensu Remediation Handler is a [Sensu Go event handler][handler]
+which enables you to build self-healing workflows in Sensu. 
+
+The Sensu Remediation Handler &ndash; and other similar "self healing"
+workflows in Sensu &ndash; combine a few Sensu features:
 
 - An "unscheduled check" configuration (i.e. a Sensu Check with the `"publish":
```

Comment:
```suggestion
- An "unscheduled check" configuration (i.e. a Sensu check with the `"publish":
```

Reply:
Just something I noticed: the other two bullet points end with a "which provides..." phrase that explains the benefit of the listed feature. Is it possible to describe the benefit of the unscheduled check configuration similarly (i.e. in a brief phrase of 5 to 10 words)?